lib/Helpers.d.ts

```diff
@@ -111,7 +123,10 @@ interface DeleteAction {
 
 type CreateAction = CreateActionOperation | [CreateActionOperation, unknown];
 type IndexAction = IndexActionOperation | [IndexActionOperation, unknown];
-type UpdateAction = [UpdateActionOperation, Record<string, any>];
```

Comment:
Does this mean the old method already supports `script` since it would accept anything with `Record<string,any>`?

Reply:
@nhtruong 
The old method accepted script input with Record<string,any> as the type, but since script was not supported in the first place, unintended queries were generated.

Reply:
@nhtruong 
The old method accepted script input with Record<string,any> as the type,
but since script was not supported in the first place, unintended queries were generated.

Reply:
Gotcha. I thought the users couldn't do script before from the title of the PR.